library/src/main/java/io/ona/kujaku/helpers/PermissionsHelper.java

```diff
@@ -9,23 +9,33 @@
 import com.karumi.dexter.Dexter;
 import com.karumi.dexter.listener.multi.MultiplePermissionsListener;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import io.ona.kujaku.utils.KujakuMultiplePermissionListener;
+import timber.log.Timber;
 
 public class PermissionsHelper {
 
     public static void checkPermissions(String TAG, Context context) {
         if (context instanceof Activity) {
+            List<String> permissions = new ArrayList<>();
+            permissions.add(Manifest.permission.ACCESS_FINE_LOCATION);
+
+            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.S) {
+                permissions.add(Manifest.permission.WRITE_EXTERNAL_STORAGE);
+            }
```

Comment:
@Lentumunai-Mark Do we have tests for this?

Reply:
@dubdabasoduba no existing tests for the class.

Reply:
Please create one